vm/src/builtins/type.rs

```diff
@@ -5,7 +5,7 @@ use super::{
 use crate::{
     builtins::{
         descriptor::{
-            DescrObject, MemberDef, MemberDescrObject, MemberGetter, MemberKind, MemberSetter,
+            DescrObject, MemberGetter, MemberKind, MemberSetter, PyMemberDef, PyMemberDescriptor,
```

Comment:
Would it make sense to also have `PyMemberGetter`, `PyMemberKind`, and `PyMemberSetter`?

Reply:
Though I am not confident of this logic,

Those types are RustPython's internal while PyMemberDef is a part of public python C-API starting with `PyMember_`

We don't use Py prefix for our internal but use it for exposed Python types. But not sure about the exact boundary yet.